src/components/modals/ConditionCreatorModal.tsx

```diff
@@ -0,0 +1,322 @@
+                            {(showNumberValue || !selectedEnumValueOption)
+                                ? <div data-testid="condition-creator-modal-dropdown-numbervalue">
+                                    <OF.Label>Number</OF.Label>
+                                    <OF.SpinButton
+                                        value={numberValue.toString()}
+                                        onBlur={(event: React.FocusEvent<HTMLInputElement>) => {
+                                            const value = parseInt(event.target.value)
+                                            if (!Number.isNaN(value)) {
+                                                setNumberValue(value)
+                                            }
+                                        }}
+                                        onDecrement={v => setNumberValue(prevValue => prevValue - 1)}
+                                        onIncrement={v => setNumberValue(prevValue => prevValue + 1)}
+                                        labelPosition={Position.bottom}
+                                        step={1}
+                                    />
+                                </div>
+                                : <OF.Dropdown
+                                    label="Enum Value"
+                                    data-testid="condition-creator-modal-dropdown-enumvalue"
+                                    selectedKey={selectedEnumValueOption.key}
+                                    options={enumValueOptions}
+                                    onChange={onChangeEnumValueOption}
+                                />}
+                        </div>
+
+                        <h2 style={{ fontWeight: OF.FontWeights.semibold as number }} className={OF.FontClassNames.large}>Existing Conditions:</h2>
+                        <div className="cl-condition-creator__existing-conditions">
+                            {conditionsUsingEntity.map(condition => {
+                                const conditionalTag = convertConditionToConditionalTag(condition, props.entities)
+                                const isActive = isConditionEqual(condition, currentCondition)
+
+                                return <React.Fragment key={conditionalTag.key}>
+                                    <div className="cl-condition-creator__existing-condition"
+                                        data-testid="condition-creator-modal-existing-condition">
+                                        {conditionalTag.name}
+                                    </div>
+
+                                    <OF.DefaultButton
+                                        data-testid="condition-creator-modal-button-use-condition"
+                                        onClick={() => onClickExistingCondition(conditionalTag.condition!)}
+                                    >
+                                        Use Condition
```

Comment:
I'm not sure I'm a fan of the big list of "Use Condition" buttons.  Why not make the condition text in to a button?

Reply:
Usability. You want user to know it's clickable and describe what it will do when clicked.
They are familiar with buttons being clickable clickable and it has label to describe what it will do when clicked.

Clickable conditions do neither of those things.

Reply:
I'm suggesting make them look like buttons 
![image](https://user-images.githubusercontent.com/6528486/66683326-0a7a1580-ec2c-11e9-94a8-9d18d4d5ee94.png)
@swadheen  - what do you think?

Reply:
That doesn't give any information about what happens when you click the button. You only solved half of the problem

And adding "use condition" in there is too noisy. Buttons should be actions like "Cancel" "Save" "Create" etc. or in this case "Use".